utils/memeUtils/memeUtils.py

```diff
@@ -74,34 +115,50 @@ async def get_random_meme(bannerized: bool) -> Tuple[bytes, Meme]:
 
     random_meme: Meme = random.choice(memes)
     meme_path = os.path.join(
-        Constants.FILE_PATHS.BANNERIZED_MEME_FOLDER if bannerized else Constants.FILE_PATHS.RAW_MEME_FOLDER,
-        f"{random_meme.uuid}.{random_meme.format.value}")
+        Constants.FILE_PATHS.BANNERIZED_MEME_FOLDER
+        if bannerized else Constants.FILE_PATHS.RAW_MEME_FOLDER,
+        f"{random_meme.uuid}.{random_meme.format.value}"
+    )
 
     with open(meme_path, 'rb') as f:
         meme_image = f.read()
 
     return meme_image, random_meme
 
 
-async def search_memes(search: str, num: int, bannerized: bool = False) -> list[tuple[bytes, Meme]]:
+async def search_memes(search: str,
+                       num: int,
+                       bannerized: bool = False) -> list[tuple[bytes,
+                                                               Meme]]:
     """
     Searches for memes containing the given search term.
     """
-    memes: list[tuple[int, Meme]] = [
-        (fuzz.token_set_ratio(search, meme.content + "\n" + meme.message), meme)
-        for meme
-        in await Meme.all()]
-
-    filtered_memes: list[tuple[int, Meme]] = [(ratio, meme) for ratio, meme in memes if ratio > 50]
+    memes: list[
+        tuple[int,
+              Meme]
+    ] = [
+        (
+            fuzz.token_set_ratio(search,
+                                 meme.content + "\n" + meme.message),
+            meme
+        ) for meme in await Meme.all()
+    ]
+
+    filtered_memes: list[tuple[int,
+                               Meme]] = [
+                                   (ratio,
+                                    meme) for ratio, meme in memes if ratio > 50
+                               ]
```

Comment:
This is really unreadable. In such cases i split up the type definition and the assignment to make it more readable, like so:

```py
number: int
number = 10
```

W: Make this more readable.